types.toml

```
@@ -33,9 +33,21 @@ uid = "5183091736401"
 title = "Types Team High Bandwidth Discussions"
 description = "Regular meeting discussing items from the Types Team roadmap"
 location = "Jitsi (https://meet.jit.si/curry-howard-ftw)"
-last_modified_on = "2024-11-30T10:39:00.00Z"
+last_modified_on = "2024-12-17T10:00:00.00Z"
```

@spastorino I'd push the `last_modified_on` a few days forward? Otherwise clients might not catch that change in time. As I also learned, see [the README](https://github.com/rust-lang/calendar?tab=readme-ov-file#how-do-i-update-an-event).

```suggestion
last_modified_on = "2024-12-20T10:00:00.00Z"
```

---

oh. too late.

---

I'm not sure if pushing too lang is good advice but I'm not sure what's the right approach here. In my case as I've added the last_modified_on for the future I wasn't able to see the update recently and I'm not sure for instance how google calendar synchronizes. If it does when you open your calendar regardless of how long you in future you place the last_modified_on if the user didn't open the calendar it won't update.

So again, not sure how it works, not sure what's the best advice to be honest.

---

> not sure how it works, not sure what's the best advice to be honest.

I think nobody really knows that. It looks more like a mix of wishful thinking and cargo culting :slightly_smiling_face: